packages/kit/test/apps/basics/test/test.js

```diff
@@ -864,7 +864,7 @@ test.describe('Actions', () => {
 			const log_promise = page.waitForEvent('console');
 			const button = page.locator(`#${id}`);
 			await button.click();
-			expect(await button.textContent()).toBe('processed'); // needed to make sure action completes
+			await expect(button).toHaveAttribute('data-processed', 'true')
```

Comment:
```suggestion
			await expect(button).toHaveAttribute('data-processed', 'true');
```

Reply:
Will this wait for it to change from `false` to `true`?

Reply:
It'll throw if it doesn't become true within a timeout, 5 secs IIRC.